### PR TITLE
Feature/maqs6.0 update documentation

### PR DESCRIPTION
**Pull request by FermJacob**

### CODE REVIEW

File: Framework/BaseAppiumTest/BaseAppiumTest.csproj [hunk 1]
```diff
@@ -1,15 +1,19 @@
-﻿  <Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net471;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <AssemblyVersion>5.8.1</AssemblyVersion>
```
Comment (TroyWalshProf) [inline Framework/BaseAppiumTest/BaseAppiumTest.csproj:5]: Would just bump these to 6.0 right now

File: Framework/BaseAppiumTest/BaseAppiumTest.csproj [hunk 2]
```diff
@@ -1,15 +1,19 @@
-﻿  <Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (TroyWalshProf) [inline Framework/BaseAppiumTest/BaseAppiumTest.csproj:2]: May add something like:

```
 <PropertyGroup>  
    <BuildVersion>6.0.0</BuildVersion>  
  </PropertyGroup>  
  <PropertyGroup>  
    <TargetFrameworks>net471;netstandard2.0;netstandard2.1</TargetFrameworks>  
    <AssemblyVersion>$(BuildVersion)</AssemblyVersion>
    <FileVersion>$(BuildVersion)</FileVersion>
    <Version>$(BuildVersion)</Version> 

```
so we update the version all at once

File: Framework/BaseAppiumTest/BaseAppiumTest.csproj [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
```
Comment (TroyWalshProf) [inline Framework/BaseAppiumTest/BaseAppiumTest.csproj:1]: Don't see it directly in hear but would remove:
PackageIconUrl
*This is deprecated

File: Framework/BaseCompositeTest/BaseCompositeTest.csproj [hunk 4]
```diff
@@ -19,7 +22,110 @@
     <PackageProjectUrl>https://github.com/Magenic/MAQS</PackageProjectUrl>
     <PackageTags>test selenium appium automation NUnit MSTest TestFramework</PackageTags>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageReleaseNotes>6.0.0 - 2020-??-??</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/Magenic/MAQS</RepositoryUrl>
+    <RepositoryType>git</RepositoryType>
+    <PackageReleaseNotes>
+      6.0.0 - 2020/05/15
```
Comment (TroyWalshProf) [inline Framework/BaseCompositeTest/BaseCompositeTest.csproj:28]: Would just start fresh with 6.0 content

Comment (FermJacob) [reply Framework/BaseCompositeTest/BaseCompositeTest.csproj:28]: This is the default Magenic.Maqs.  Do we want to keep the history for this package or start fresh?

**Comment by FermJacob** [comment]
@TroyWalshProf Updated with comments.  Let me know if you find anything else

File: Framework/BaseCompositeTest/BaseCompositeTest.csproj [hunk 5]
```diff
@@ -1,25 +1,39 @@
...
+      Release of MAQS 6.0
+      - Updated to individual NuGet packages
+      - Changes to Lazy Elements
```
Comment (TroyWalshProf) [inline Framework/BaseCompositeTest/BaseCompositeTest.csproj:34]: Maybe we shouldn't put sub module changes in this log or we want all sub module changes - Thoughts?

File: Framework/SpecFlowExtension/SpecFlowExtension.csproj [hunk 6]
```diff
@@ -36,6 +62,10 @@
     <PackageReference Include="Utf8Json" version="1.3.7" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="..\MAQS.jpg" Pack="true" PackagePath="" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\BaseAppiumTest\BaseAppiumTest.csproj" />
```
Comment (TroyWalshProf) [inline Framework/SpecFlowExtension/SpecFlowExtension.csproj:70]: We would likely want this to depend on the composite project

**Pull request merged**